cody/Index 2.0.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="./style.css" />
+  <title>Page Titlez</title>
```

Comment:
This page title is what is displayed at the top of the browser. This should be something like "Patient Viewer" of "KMPT | Patient"